vpr/src/route/routing_predictor.cpp

```diff
@@ -63,12 +63,8 @@ float covariance(std::vector<size_t> x_values, std::vector<float> y_values, floa
     return cov;
 }
 
-float RoutingPredictor::get_slope() {
-    if (iterations_.size() > min_history_) {
-        auto model = fit_model(iterations_, iteration_overused_rr_node_counts_, history_factor_);
-        return model.get_slope();
-    }
-    return -1;
+float RoutingPredictor::get_slope() const {
```

Comment:
Should comment the caching of slope (you mentioned it n the meeting).

Reply:
Added comment.